plugins/poll/assets/javascripts/discourse/templates/modal/poll-ui-builder.hbs

```diff
@@ -17,6 +17,15 @@
                     valueAttribute="value"}}
       </div>
 
+      {{#unless isNumber}}
+        <div class="input-group poll-select">
+          <label class="input-group-label">{{i18n 'poll.ui_builder.poll_chart_type.label'}}</label>
+          {{combo-box content=pollChartTypes
+                      value=chartType
+                      valueAttribute="value"}}
```

Comment:
I thought `value` is the default valueAttribute? Am I mistaken?

Reply:
Other combo-boxes in the same file have it set like this. I followed suite. I can fix it in the other places as well.